server/planning/__init__.py

```diff
@@ -114,6 +118,49 @@ def init_app(app):
     app.client_config['street_map_url'] = get_street_map_url(app)
     app.client_config['max_multi_day_event_duration'] = get_event_max_multi_day_duration(app)
 
+    # Set up Celery task options
+    if not app.config.get('CELERY_TASK_ROUTES'):
+        app.config['CELERY_TASK_ROUTES'] = CTR
+
+    if not app.config.get('CELERY_TASK_ROUTES').get('planning.flag_expired'):
+        app.config['CELERY_TASK_ROUTES']['planning.flag_expired'] = {
+            'queue': celery_queue('expiry'),
+            'routing_key': 'expiry.planning'
+        }
+
+    if not app.config.get('CELERY_BEAT_SCHEDULE'):
+        app.config['CELERY_BEAT_SCHEDULE'] = CBS
+
+    if not app.config.get('CELERY_BEAT_SCHEDULE').get('planning:expiry'):
+        app.config['CELERY_BEAT_SCHEDULE']['planning:expiry'] = {
+            'task': 'planning.flag_expired',
+            'schedule': crontab(minute='0')  # Runs once every hour
+        }
+
+    # Create 'type' required for planning module if not already preset
+    with app.app_context():
+        vocabulary_service = superdesk.get_resource_service('vocabularies')
+        types = vocabulary_service.find_one(req=None, _id='type')
+        items = types.get('items') or []
+        if types:
+            added_types = []
+            type_names = [t['name'] for t in items]
+
+            planning_type_list = [
+                {"is_active": True, "name": "planning item", "qcode": "planning"},
+                {"is_active": True, "name": "event", "qcode": "event"},
+                {"is_active": True, "name": "Featured Stories", "qcode": "planning_featured"}
+            ]
+
+            for type in planning_type_list:
```

Comment:
`type` is an in-built function in python and should be avoided for class/variable names.